apps/web/app/(api-routes)/escrow/initialize/route.ts

```
@@ -0,0 +1,88 @@
+import { NextRequest, NextResponse } from "next/server";
+import { createClient } from "@supabase/supabase-js";
+import { validateEscrowInitialization } from "~/lib/validators/escrow";
+import { initializeEscrowContract } from "~/lib/stellar/escrow";
+import { EscrowInitialization } from "~/lib/types/escrow";
```

Biome (1.9.4)

[error] All these imports are only used as types.

Importing the types with import type ensures that they are removed by the compilers and avoids loading unnecessary modules.
Safe fix: Use import type.


(lint/style/useImportType)